src/System.Device.Gpio/CompatibilitySuppressions.xml

```diff
@@ -13,6 +13,12 @@
     <Left>lib/netstandard2.0/System.Device.Gpio.dll</Left>
     <Right>lib/net6.0-windows10.0.17763/System.Device.Gpio.dll</Right>
   </Suppression>
+  <Suppression>
+    <DiagnosticId>CP0002</DiagnosticId>
```

Comment:
I wouldn't expect this to show up as we should have the finalizer in both places. Can you please check why this is needed?

Reply:
I have no clue, but looks like a bug of the tool to me. We have _added_ a finalizer, so why should there be a warning that it's now missing? I can't see why this would be a breaking change, anyway.

Reply:
I verified the generated IL. The finalize method is also present in the netstandard2.0 assembly.